File: SEPTEMBER 24/05 September/Missing in Array gfg.cpp
class Solution {
  public:

    // Note that the size of the array is n-1
    int missingNumber(int n, vector<int>& arr) {
        long long total = (n*(n+1))/2;
        for(int num:arr)
            total -= num;
        return total;
    }
};

File: APRIL 2025/21 April/Missing in Array gfg.java

class Solution {
    int missingNum(int arr[]) {
        // code here
        int N = arr.length+1;
        long expectedSum = (1L*N*(N+1))/2, actualSum = 0L;
        for(int num:arr)
            actualSum += num;
        return (int) (expectedSum-actualSum);
    }
}
